### PR TITLE
LPS-147456

### PR DESCRIPTION
**Pull request by rodrigopaulino**
Related Issue: https://issues.liferay.com/browse/LPS-147456

### CODE REVIEW

**Comment by liferay-forms** [comment]
ci:test:relevant

**Comment by liferay-forms** [comment]
Hey @rodrigopaulino, can you please check if these failures are related to your changes? Thanks

**Comment by rodrigopaulino** [comment]
ci:test:relevant

File: modules/apps/dynamic-data-mapping/dynamic-data-mapping-service/src/main/java/com/liferay/dynamic/data/mapping/service/impl/DDMFieldLocalServiceImpl.java [hunk 1]
```diff
@@ -533,10 +538,15 @@ private void _collectDDMFieldInfos(
 					String languageId = LanguageUtil.getLanguageId(
 						entry.getKey());
 
+					String sanitizedValue = SanitizerUtil.sanitize(
+						0, 0, 0, DDMFieldAttribute.class.getName(), 0,
+						ContentTypes.TEXT_HTML,
+						_sanitize(ddmFormField.getType(), entry.getValue()));
+
```
Comment (carolmariaabb) [inline modules/apps/dynamic-data-mapping/dynamic-data-mapping-service/src/main/java/com/liferay/dynamic/data/mapping/service/impl/DDMFieldLocalServiceImpl.java:545]: @rodrigopaulino Why do we need to call ` SanitizerUtil.sanitize` twice? One here and one in `private String _sanitize(String ddmFormFieldType, String value)`

Comment (rodrigopaulino) [reply modules/apps/dynamic-data-mapping/dynamic-data-mapping-service/src/main/java/com/liferay/dynamic/data/mapping/service/impl/DDMFieldLocalServiceImpl.java:545]: We don't! I'll fix this right away. Thanks, @carolmariaabb

**Comment by rodrigopaulino** [comment]
ci:stop

**Comment by rodrigopaulino** [comment]
ci:test:sf

**Comment by liferay-forms** [comment]
Hey @rodrigopaulino, can you please check if these failures are related to your changes? Thanks

File: modules/apps/dynamic-data-mapping/dynamic-data-mapping-service/src/main/java/com/liferay/dynamic/data/mapping/service/impl/DDMFieldLocalServiceImpl.java (hunk 1 above)
Comment (rodrigopaulino) [reply modules/apps/dynamic-data-mapping/dynamic-data-mapping-service/src/main/java/com/liferay/dynamic/data/mapping/service/impl/DDMFieldLocalServiceImpl.java]: Done, @carolmariaabb

**Comment by liferay-forms** [comment]
Hey @rodrigopaulino, can you please check if these failures are related to your changes? Thanks

**Comment by rodrigopaulino** [comment]
The failure is related to the PR.

**Comment by rodrigopaulino** [comment]
The solution is not complete. Apparently, Liferay's default sanitizer removes href attributes from tags that were expected to have. Example: video, audio, image links.

**Pull request closed without merging**